lib/util.js

```diff
@@ -84,10 +92,88 @@ exports.isIndex = (path) => {
   }
 };
 
+exports.getInlineCss = (isDist) => {
+  if (isDist) {
+    let files = [];
+    try { files = fs.readdirSync(`${__dirname}/../dist`); } catch (e) {}
+    let cssFile = files.find(f => f.match(/^styles\..+\.bundle\.css$/));
+    if (cssFile) {
+      return `<style>${fs.readFileSync(`${__dirname}/../dist/${cssFile}`)}</style>`;
+    }
+  } else {
+    return '';
+  }
+}
+
+
+exports.getInlineJs = (isDist) => {
+  if (isDist) {
+    let files = [];
+    try { files = fs.readdirSync(`${__dirname}/../dist`); } catch (e) {}
+    let inlineFile = files.find(f => f.match(/^inline\..+\.bundle\.js$/));
+    if (inlineFile) {
+      return `<script>${fs.readFileSync(`${__dirname}/../dist/${inlineFile}`)}</script>`;
+    }
+  } else {
+    return '<script src="inline.bundle.js"></script>';
+  }
+}
+
+exports.getLoadingMarkup = (isDist) => {
+  if (isDist) {
+    return inlineLoadingPlayer("../dist/app/shared/player/player.component.css", "../dist/app/shared/player/player.component.html");
+  } else {
+    return inlineLoadingPlayer("../src/app/shared/player/player.component.css", "../src/app/shared/player/player.component.html");
+  }
+}
+
 /**
  * Spawn a process to run ng serve
  */
 exports.ngServe = (port) => {
   let ng = `${__dirname}/../node_modules/.bin/ng`;
   return spawn(ng, ['serve', '--port', port], {stdio: 'inherit'});
 };
+
+function inlineLoadingPlayer(cssPath, htmlPath) {
+  const cssContent = fs.readFileSync(`${__dirname}/${cssPath}`).toString('utf-8');
+  const mediaQueries = mediaQ(cssContent).replace(/;/g,' !important;').replace(/\s+/g,' ').replace(/\s?([\{\},:\(\)])\s?/g, '$1').replace(/;\s?\}/g, '}');
+  return inlineCss(`<style>${cssContent}</style>
+    <div class="loadingRender">${fs.readFileSync(`${__dirname}/${htmlPath}`)}</div>
+    `, {url: '/'}).then((markup) => {
+    const $ = cheerio.load(markup, {normalizeWhitespace: true});
+    scrub($, $('.loadingRender').children());
+    return `<style>${mediaQueries}</style>${$('.loadingRender').html()}`;
```

Comment:
Okay, so you're inlining the normal css from `player.component.css`, and throwing the media queries into the `<style>`.  I get it.  I get things.